StyleSheet.css

```diff
@@ -21,6 +21,62 @@ nav {
   padding: 6px 24px;
 }
 
+.nav-wrapper {
+  position: fixed;
+  left: -100%;
+  top: 0;
+  height: 100vh;
+  width: 100vw;
+  -moz-transition: all 0.3s ease-in;
+  -ms-transition: all 0.3s ease-in;
+  -o-transition: all 0.3s ease-in;
+  -webkit-transition: all 0.3s ease-in;
+  transition: all 0.3s ease-in;
+}
+
+.nav-wrapper.open {
+  left: 0;
+}
+
+.nav-bg {
+  background-color: #6070ffdc;
+  -webkit-backdrop-filter: blur(20px);
+  backdrop-filter: blur(20px);
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  inset: 0;
+}
```

Comment:
- [ ] Kindly make sure your mobile menu design sticks as much as possible to the one from the Figma template. I suggest you improve your `blur effect`, `fonts on the text`, `spacing around and between elements`,... You can check all details from Figma to improve your design

| Figma design | Your Design |
| ------------------- | ------------------- |
| ![Screenshot from 2022-12-19 22-38-18](https://user-images.githubusercontent.com/83042742/208527401-87b55d73-e767-4ba1-ac2b-95e6fa6fd76d.png) | ![Screenshot from 2022-12-19 23-30-05](https://user-images.githubusercontent.com/83042742/208528088-331647fb-42c6-4ee9-900c-156b3f003e20.png) |